doc/implementers-guide/implementers-guide.adoc

```diff
@@ -684,4 +684,168 @@ creating a managed binary is itself a secondary compilation and optimization
 step. The difference is that performing this step is a system rather than a
 developer responsibility.
 
+== Configuration
+Each ODP implementation will choose various sizes, limits, and similar
+internal parameters that are well matched to its design and platform
+capabilities. However, it is often useful to expose at least some of these
+parameters and allow applications to select specific values to use either
```

Comment:
I'd say: "... allow users to select ... ", instead of "applications to select". As user builds and launches applications with certain config values.